linter.py

```diff
@@ -30,11 +31,39 @@ class ESLint(NodeLinter):
         r'(?:(?P<error>Error)|(?P<warning>Warning)) - '
         r'(?P<message>.+)'
     )
+    config_fail_regex = re.compile(
+        r'^Cannot read config file: (.*?)\r?\n'
+        r'Error: (?:JS-YAML: )?(.*?):?\r?\n',
+        re.MULTILINE
+    )
+    crash_regex = re.compile(
+        r'^(.*?)\r?\n\w*Error: \1',
+        re.MULTILINE
+    )
     line_col_base = (1, 0)
     selectors = {
         'html': 'source.js.embedded.html'
     }
 
+    def find_errors(self, output):
+        """
+        Parses errors from linter's output
+
+        We override this method to handle parsing eslint crashes
+        """
+
+        match = self.config_fail_regex.match(output)
+        if match:
+            msg = "%s in %s" % match.group(2, 1)
```

Comment:
I suppose message `Cannot read config file: /path/.eslintrc` will be more relevant, full error message is trimmed by output and is not clear, user can run eslint in cli to get specific location of error.